### ISSUE TITLE
With react-native 0.74 min android sdk version needs to updated

### ISSUE THREAD

**Issue by jkdowdle** [mixpanel/mixpanel-react-native#252]
## Motivation

I was attempting to upgrade my project to use the react-native@0.74.1 and when I tried to create an android build I ran into this error message indicating that the libraries have a differing min sdk version (21 vs 23)

It seems like I could use the expo / non-native version but I would like to continue to use the native.

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':mixpanel-react-native:processDebugAndroidTestManifest'.
> Manifest merger failed : uses-sdk:minSdkVersion 21 cannot be smaller than version 23 declared in library [com.facebook.react:react-android:0.74.1] /Users/joshuadowdle/.gradle/caches/transforms-4/fc4dd7a54b1a3c5197151f78787e8c22/transformed/jetified-react-android-0.74.1-debug/AndroidManifest.xml as the library might be using APIs not available in 21
        Suggestion: use a compatible library with a minSdk of at most 21,
                or increase this project's minSdk version to at least 23,
                or use tools:overrideLibrary="com.facebook.react" to force usage (may lead to runtime failures)
```

**Comment by WrathChaos** [comment]
I have the same issue, will someone fix it?

**Comment by zihejia** [comment]
Thanks @jkdowdle @WrathChaos for bringing it up. It's fixed in 3.0.6!
